forecasting/src/autogluon/forecasting/models/presets.py

```diff
@@ -0,0 +1,199 @@
+import copy
+import logging
+from typing import Union, Dict
+
+import autogluon.core as ag
+
+from .abstract import AbstractForecastingModel
+from .abstract.abstract_forecasting_model import AbstractForecastingModelFactory
+from .gluonts import (
+    DeepARModel,
+    MQCNNModel,
+    ProphetModel,
+    SimpleFeedForwardModel,
+    AutoTabularModel,
+)
+
+logger = logging.getLogger(__name__)
+
+
+MODEL_TYPES = dict(
+    MQCNN=MQCNNModel,
+    SimpleFeedForward=SimpleFeedForwardModel,
+    DeepAR=DeepARModel,
+    AutoTabular=AutoTabularModel,
+    Prophet=ProphetModel,
+)
+DEFAULT_MODEL_NAMES = {v: k for k, v in MODEL_TYPES.items()}
+DEFAULT_MODEL_PRIORITY = dict(
+    MQCNN=50,
+    SimpleFeedForward=30,
+    DeepAR=50,
+    Prophet=50,
+    AutoTabular=10,
+)
+DEFAULT_CUSTOM_MODEL_PRIORITY = 0
+
+
+def get_default_hps(key, prediction_length):
+    default_model_hps = {
+        "toy": {
+            "SimpleFeedForward": {
+                "epochs": 5,
+                "num_batches_per_epoch": 10,
+                "context_length": 5,
+            },
+            "MQCNN": {"epochs": 5, "num_batches_per_epoch": 10, "context_length": 5},
+            "DeepAR": {"epochs": 5, "num_batches_per_epoch": 10, "context_length": 5},
+        },
+        "toy_hpo": {
+            "SimpleFeedForward": {
+                "epochs": 5,
+                "num_batches_per_epoch": 10,
+                "context_length": ag.Int(5, 25),
+            },
+            "MQCNN": {
+                "epochs": 5,
+                "num_batches_per_epoch": 10,
+                "context_length": ag.Int(5, 25),
+            },
+            "DeepAR": {
+                "epochs": 5,
+                "num_batches_per_epoch": 10,
+                "context_length": ag.Int(5, 25),
+            },
+        },
+        "default": {
+            "SimpleFeedForward": {},
+            "MQCNN": {},
+            "DeepAR": {},
+            # "AutoTabular": {} # AutoTabular model is experimental.
+        },
+        "default_hpo": {
```

Comment:
I guess the search space for now is tentative and the final search space definition will come in the future?

Reply:
I didn't think these were bad defaults. Should we think about others?

Reply:
I don't know they are good or bad but I know their implication is significant. Want to call out so that we put enough attention to it. Especially given that we don't do multi-layer stacking in the beginning where the model is just trained with default HPs without any HPO.